scripts/predict.py

```diff
@@ -1,45 +1,59 @@
+from __future__ import print_function
 import json
 import os
+import re
 import pickle
 import sys
+import click
 from os.path import dirname, abspath
 sys.path.append(dirname(dirname(abspath(__file__))))
 from pkspace.utils.loaders import PKSpaceLoader # noqa
-import click # noqa
 
 
 @click.command()
 @click.option('--mask_path', required=True,
-              help='Adress of json file of the mask')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path of json file of the mask')
 @click.option('--picture_path', required=True,
-              help='Adress of picture to be predicted')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path of picture to be predicted')
 @click.option('--model_path', required=True,
-              help='Path where pretrained model')
+              type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+              help='Path where pretrained model is located')
 @click.option('--output', default=None,
-              help='Name of output file for predicted output')
-def load_and_predict(mask_path, picture_path, model_path, output):
-    for file in (mask_path, picture_path, model_path):
-        if not os.path.exists(file):
-            print(file, "Path does not exist")
-            return
+              help='Name of file for predicted output')
+@click.option('--img_suffix', default='.png',
+              help='Suffix of predicted image')
```

Comment:
@Adman This should be probably removed and replace by simple check for extension in the outputs name and then replacing the extension or if extension is `None` then adding `_out.json` or something like that.